### ISSUE TITLE
Enhancement: Implementation of Queue

### ISSUE THREAD

**Issue by AshKetshup** [borisdayma/dalle-mini#224]
To avoid the multiple requests, maybe the implementation of a queue might help the flow of the website

**Comment by NomsterDude** [comment]
I agree with this idea. It theoretically shouldn't affect the functionality/wait for the website, it should only add better organization.
